qcweb/plotting.py

```diff
@@ -16,15 +16,17 @@ def plot_demo(data_frame):
     # matplot/ seaborn style setting
     sns.set_style('whitegrid')
     mpl.rcParams['patch.force_edgecolor'] = True
-    sns.set(rc={'figure.figsize':(5.0, 5.0)})
+    sns.set(rc={'figure.figsize': (5.0, 5.0)})
 
     # seaborn countplot
     p1 = sns.countplot(x='Group', data=data_frame, palette='coolwarm')
 
     # rotate xticklabels to prevent the labels being overlapped
+    # use a semicolon to suppress the output of a final function
```

Comment:
Suppressing output with a semicolon is an IPython concept.

Semicolons should be avoided in Python files.